erpnext/manufacturing/doctype/work_order/work_order.js

```diff
@@ -646,15 +646,26 @@ erpnext.work_order = {
 
 	show_prompt_for_qty_input: function(frm, purpose) {
 		let max = this.get_max_transferable_qty(frm, purpose);
+		let label = __('Qty for {0}', [purpose]);
+		if(frm.doc.manufacturing_type === "Process" && purpose === "Manufacture"){
```

Comment:
if(frm.doc.manufacturing_type === "Process"){
label = __('Raw Material Qty for {0}', [purpose]);